pkg/cgroups/blkio.go

```diff
@@ -37,7 +37,7 @@ func (c *blkioHandler) Create(ctr *CgroupControl) (bool, error) {
 
 // Destroy the cgroup
 func (c *blkioHandler) Destroy(ctr *CgroupControl) error {
-	return os.Remove(ctr.getCgroupv1Path(Blkio))
+	return rmDirRecursively(ctr.getCgroupv1Path(Blkio))
```

Comment:
Why can't you use
os.RemoveAll(ctr.getCgroupv1Path(Blkio))

Reply:
`os.RemoveAll()` would not work with cgroups.  It will try to remove all files, while we can remove just  the directories.